openhtf/plugs/usb/__init__.py

```diff
@@ -52,9 +72,21 @@ def _open_usb_handle(**kwargs):
   Returns:
     Instance of UsbHandle.
   """
+
+  serial = None
   if conf.Config().usb_server:
     return None
   else:
+    remote_usb = conf.Config().remote_usb
```

Comment:
The usb_server config key is superseded by the remote_usb key. I'm thinking we should get rid of usb_server, and assume a local connection unless remote_usb is configured.